CHANGELOG.md

```diff
@@ -14,9 +14,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 <!--
 Do *NOT* manually add changelog entries here!
 This changelog is managed by Towncrier and is built at release time.
-See https://ansible-navigator.readthedocs.io/en/latest/contributing.html#adding-change-notes-with-your-prs
+See https://ansible-navigator.rtfd.io/en/latest/contributing/guidelines/#adding-change-notes-with-your-prs
```

Comment:
Can we switch to the full URL (readthedocs) instead of RTFD?  We try not to use the acronym since it might be misinterpreted to mean something we don't want it to :)